projects/observability/src/shared/dashboard/data/graphql/explore/explore-cartesian-data-source.model.ts

```diff
@@ -65,16 +64,19 @@ export class ExploreCartesianDataSourceModel extends GraphQlDataSourceModel<Metr
         (selection): selection is RequireBy<ExploreSpecification, 'aggregation'> => selection.aggregation !== undefined
       );
 
-    if (request.interval && !request.groupBy) {
+    const groupByKeys = request.groupBy?.keys || [];
```

Comment:
`??` ?

Reply:
Old habits - although they're identical in this case, will update.